src/resource-stream.ts

```diff
@@ -47,7 +47,7 @@ export class ResourceStream<T> extends Transform implements ResourceEvents<T> {
     this._requestsMade = 0;
     this._resultsToSend = args.maxResults === -1 ? Infinity : args.maxResults!;
   }
-  // tslint:disable-next-line:no-any
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
I think this is fine for now, I would rather that these PRs turn into as minimal of a refactor as possible, even though we might want to come back and make this a more extensive refactor.